packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.6.13
+
+* (Depend on `@lionweb/core` and `@lionweb/utilities` version 0.6.12.)
```

Comment:
Why the parenthesis?

My understanding is that we typically list features and this is more of a comment so we distinguish it by using the parenthesis. I would however consider removing the parenthesis and the bullet point. We could also make it in italic.

Reply:
Good point! We could list it as a separate, non-list characteristic of each release: just mentioning the relevant (and in most cases: only) upstream dependencies.